### PR TITLE
tickets/SITCOM-1441

### PR DESCRIPTION
**Pull request by Kelks19**
I have created two pages for the same procedure-- one for the MTRotator and one for MTHexapods. 

Path: Simonyi/Non-Standard-Operations/MTCS/MTHexRot

### CODE REVIEW

**Review by kpenaramirez — COMMENTED** [review]
Thank you for the documents. I used [this document](https://confluence.lsstcorp.org/pages/viewpage.action?pageId=254444486) to guide my revision, now that Hexapods and Rotator are separated it is more readable. I understand it as a high level description of different levels of recovery that is why I think the final warning on the MTRotator should be move to a more detailed and focused troubleshooting procedure. 
@isotuela : I tried to follow the login instructions on the vaults and in general terms in the document are correct. Nevertheless, the vault names are confusing (e.g. M2Hexapod PXI, M2HexapodPXI/DriveNetBooter...).

File: Simonyi/Non-Standard-Operations/MTCS/MTHexRot/MTHex-PXI-Controller-Reboot.rst [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+    **Only proceed with a hard reboot, if the EUI control connection remains unsucessful after 
+    a** :ref:`soft reboot <MTRot-PXI-Controller-Reboot-Soft-Reboot>`.
+
+    This method involves cutting power to the PXI and drives and should only be used as a last resort 
+    due to the potential risks.
+
+    If a power shutdown is scheduled, you can proceed until step 2 before the power on.
+
+
+#.  **Login into Utilities cabinet *Power Distribution Unit* (*PDU*) or MTM2Hexapod *netbooter* **:
```
[the remark this answers is not in the document]
Comment (Kelks19) [reply Simonyi/Non-Standard-Operations/MTCS/MTHexRot/MTHex-PXI-Controller-Reboot.rst:206]: corrected! it seems that restructured text dies not allow bold and italics together

File: Simonyi/Non-Standard-Operations/MTCS/MTHexRot/MTHex-PXI-Controller-Reboot.rst [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+    Depending on the which subsytem you are rebooting, you need to follow either A or B
+    
+    A.  For MTCamHexapod, while in the LSST-WAP or summit VPN, 
+        connect to https://tea-pdu01.cp.lsst.org/ using the credentials stored in the *Operators vault* 
+        of *LSST 1Password* as *PDU Utilities Cabinet*. Click on :guilabel:`Outlets` on the left hand 
+        side menu to open the outlets screen. The description of each outlet can be found here.
```
[the remark this answers is not in the document]
Comment (Kelks19) [reply Simonyi/Non-Standard-Operations/MTCS/MTHexRot/MTHex-PXI-Controller-Reboot.rst:212]: The page is yet to be created!

**Review by kpenaramirez — APPROVED** [review]
Hi Kshitija,
Thank you for including the comments. Agreed with the changes.

**Review by Slimleashma — COMMENTED** [review]
Looks good to me.

**Pull request merged**